setup.py

```diff
@@ -19,7 +19,9 @@
         'dulwich',
         'PyGithub',
         'future',
-        'speedtest-cli'
+        'speedtest-cli',
+        'imageio',

```

Comment:
@manparvesh I added it because it is being used in the gif module. When I tried using the program, I got an error since the package was not installed. So I thought I would just add it to save others from dealing with the same thing.

Reply:
Oh okay. Then I guess it's okay